etc/requirements_build.in

```diff
@@ -0,0 +1,2 @@
+Cython>=0.25.2
```

Comment:
I think a good next step after merging this would be to move these into a `pyproject.toml`.